cmd/ebrelayer/txs/symbol_translator.go

```diff
@@ -0,0 +1,47 @@
+package txs
+
+import (
+	"encoding/json"
+	"github.com/Sifchain/sifnode/x/utilities"
```

Comment:
Is utilities a cosmos module? I don't think this is the place for a utility package, especially not such a generic name.

Reply:
Moved these.